### PR TITLE
Marker missing for edit orders

### LINKED ISSUE #2034: Marker missing for edit orders
@lebaphi created a fix for showing the geoman marker for drawing  new points.

When we edit an existing shape, it doesn't show an marker - the icon isn't found.

I've tried to [copy](https://github.com/serge-web/serge/pull/2033/commits/78f667a12ecbf87f2e7a78403d5d555099d8b56e) the code, but it still doesn't work.

https://user-images.githubusercontent.com/1108513/200938014-9b793104-4209-40e3-b505-76eeefd09068.mp4

Steps to reproduce:
* open storybook:  https://serge-sql.herokuapp.com/storybook/?path=/story/local-organisms-planningchannel--default
* open `orders` tab
* expand to view all rows
* find row with `point` in the `Activity`
* expand the row
* click on the `Edit` button in `Location`
* you'll see the marker isn't present

### PR DESCRIPTION
**Pull request by lebaphi**
Fixes #2034

### CODE REVIEW

**Comment by lebaphi** [comment]
Hi @IanMayo , I think this solution will fix the issue of marker icon on OrderDrawing component too, please help me to verify it :D

File: packages/components/src/local/organisms/support-panel/helpers/customise-date.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const getDateSlots = (gameTime: number, gameTurnTime: TurnLengthType): Ar
   const res: TimeSlot[] = []
   const formatStr = smallSteps ? shortFormat : longFormat
   for (let time = gameTime; time < finishTime; time += timeStep) {
```
Comment (IanMayo) [inline packages/components/src/local/organisms/support-panel/helpers/customise-date.ts]: oh dear, @lebaphi - I've made lots of use of `moment` in the last few days.   Was this to overcome a specific issue? I may have to apply the fix in lots of places.

**Comment by IanMayo** [comment]
The fix works for "edit orders". 

But, "draw new orders" is broken:
- run game
- log in as `Mar 1`
- click on button to create new orders
- select `Maritime` - `Precision Strike`
- Draw route in (line)
- The marker isn't shown for `Target Location`

**Comment by lebaphi** [comment]
okay I will check this 👍🏻

**Comment by lebaphi** [comment]
Hi @IanMayo , I reverted the changes for OrderDrawing component, I think it works now 😂

**Review by IanMayo — APPROVED** [review]
Fixed!

File: packages/components/src/local/organisms/support-panel/helpers/customise-date.ts (hunk 1 above)
Comment (lebaphi) [reply packages/components/src/local/organisms/support-panel/helpers/customise-date.ts]: Hi @IanMayo, I think we should use moment.utc(...) for this one
```
const tNow = moment.utc(time)
```
the unit test failed on my machine 😄

**Pull request merged**